releasing/update-manifests-images

```diff
@@ -0,0 +1,200 @@
+        "kustomization": "components/admission-webhook/manifests/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/admission-webhook",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/admission-webhook",
+            },
+        ],
+    },
+    {
+        "name": "Central Dashboard",
+        "kustomization": "components/centraldashboard/manifests/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/central-dashboard",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/central-dashboard",
+            },
+        ],
+    },
+    {
+        "name": "Jupyter Web App",
+        "kustomization": "components/crud-web-apps/jupyter/manifests/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/jupyter-web-app",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/jupyter-web-app",
+            },
+        ],
+    },
+    {
+        "name": "Tensorboard Web App",
+        "kustomization": "components/crud-web-apps/tensorboards/manifests/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/tensorboards-web-app",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/tensorboards-web-app",
+            },
+        ],
+    },
+    {
+        "name": "Volumes Web App",
+        "kustomization": "components/crud-web-apps/volumes/manifests/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/volumes-web-app",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/volumes-web-app",
+            },
+        ],
+    },
+    {
+        "name": "Notebook Controller",
+        "kustomization": "components/notebook-controller/config/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/notebook-controller",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/notebook-controller",
+            },
+        ],
+    },
+    {
+        "name": "Tensorboard Controller",
+        "kustomization": "components/tensorboard-controller/config/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/tensorboard-controller",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/tensorboard-controller",
+            },
+        ],
+    },
+    {
+        "name": "Profile Controller",
+        "kustomization": "components/profile-controller/config/base/kustomization.yaml",
+        "images": [
+            {
+                "name": "public.ecr.aws/j1r0q0g6/notebooks/profile-controller",
+                "newName": "public.ecr.aws/j1r0q0g6/notebooks/profile-controller",
+            },
+        ],
+    },
+    {
+        "name": "KFAM",
```

Comment:
Small nit, this name stands out compared to the rest. People not familiar with the code might now know what `KFAM` means. Not a blocker for the PR but `Access Management` might be better to use.

Reply:
Sounds reasonable